### PR TITLE
Rhornung67/docs improve

### PR DESCRIPTION
**Pull request by rhornung67**
Proofread user guide and clean up the text.

### CODE REVIEW

File: docs/sphinx/user_guide/index.rst [hunk 1]
```diff
@@ -15,26 +15,26 @@ The RADIUSS Shared CI project contains an *automated CI infrastructure based
...
    :alt: RADIUSS Shared CI infrastructure in a project repository.
    :align: center
 
-The figure illustrates the various software packages involved (orange boxes)
-and their relationships to a project that uses them. 
+   The figure illustrates the various software packages involved (orange boxes)
+   and their relationships to a project that uses them. 
 
-Typically, a project has two Git submodules: 
+A project that uses the shared CI framework, has two Git submodules typically:
```
Comment (adrienbernede) [inline docs/sphinx/user_guide/index.rst:30]:
```suggestion
A project that uses the shared CI framework typically has two Git submodules: 
```

File: docs/sphinx/user_guide/setup_ci.rst [hunk 2]
```diff
@@ -47,12 +53,15 @@ With a centralized CI configuration that is shared by projects, we create an int
 The short version
 =================
 
+The following sequence of steps and commands summarizes the process of 
+integrating the RADIUSS Shared CI infrastructure into  your project.
```
Comment (adrienbernede) [inline docs/sphinx/user_guide/setup_ci.rst:57]:
```suggestion
integrating the RADIUSS Shared CI infrastructure into your project.
```

**Review by adrienbernede — COMMENTED** [review]
LGTM !

**Review by adrienbernede — APPROVED** [review]

**Comment by rhornung67** [comment]
@adrienbernede thanks for the changes and merging.

**Pull request merged**